person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+  end
```

Comment:
- [ ] You need to define an instant variable `@parebt_permission` in the constructor so that you are able to use it in the class methods such as `can_use_services?` method.